services/src/main/scala/cromwell/pipeline/service/ProjectService.scala

```diff
@@ -64,8 +80,11 @@ object ProjectService {
             active = true
           )
         projectVersioning.createRepository(localProject).flatMap {
-          case Left(exception) => Future.successful(Left(exception))
-          case Right(project)  => projectRepository.addProject(project).map(_ => Right(project))
+          case Left(_) => Future.failed(InternalError("Failed to create project due to unexpected internal error"))
```

Comment:
Should we treat versioning exceptions differently? Or at least make exception message more informative?